ui_framework/src/components/header/_header_logo.scss

```diff
@@ -0,0 +1,13 @@
+@include component('kuiHeaderLogo') {
+  position: relative;
+  height: $kuiHeaderChildSize;
+  padding: $kuiSize $kuiSizeXL;
+  display: inline-block;
+  vertical-align: middle;
+
+  @include child('icon') {
+    height: $kuiSizeXL !important;
```

Comment:
It's the default for when no size is specified. To remove the `!important`, we have to remove the default size, which is fine with me.